docker/Dockerfile

```diff
@@ -1,11 +1,7 @@
-FROM rocker/r-ver:4.3.3
+FROM bioconductor/r-ver:3.18
 LABEL maintainer="ccdl@alexslemonade.org"
 LABEL org.opencontainers.image.source https://github.com/AlexsLemonade/scpcaTools
 
-COPY scripts/install_scpca_deps.sh .
-
-RUN bash ./install_scpca_deps.sh
```

Comment:
Why don't we need these anymore? I would think for the full image here we would?